pages/common/starship-init.md

```diff
@@ -0,0 +1,20 @@
+# starship init
+
+> Print the integration code for the infinitely customizable prompt for any shell.
+> More information: <https://starship.rs>.
+
+- Display the subcommand help:
```

Comment:
```suggestion
- Print the subcommand help:
```

The other 2 have print (desc and second command) however this doesn't and

Reply:
This is getting ridiculous, I have to admit. 
Maybe I (mainly me) should stop changing these unless required, however I feel this one is warranted

Reply:
🤔 we use `display` in a lot of pages, `print` is used just in 3 pages.

Reply:
We'll leave it as display